hack/kubernetes/allinone.sh

```diff
@@ -6,8 +6,60 @@
 set -o errexit
 set -o nounset
 
-KUBERNETES_VERSION_UBUNTU="1.9.4-00"
-KUBERNETES_VERSION_CENTOS="1.9.4"
+echo "----------------------------------"
+echo "Please choose the version of kubernetes"
```

Comment:
How about a extra colon?
echo "Please choose the version of kubernetes" -> echo "Please choose the version of kubernetes:"